src/cosmosdb-preview/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.11.0
+++++++
+* Create and manage Role Definitions and User Definitions for enforcing data plane RBAC on Cosmos DB MongoDB accounts
```

Comment:
Besides updating the release history, the `VERSION` defined in `setup.py` should also be updated to make it really work.